scripts/dbMaintenance/removeOrphanFiles.py

```diff
@@ -23,8 +26,9 @@
 connString = "mongodb://"+ userName + ":" + password +"@"+mongoURL + ":" + mongoPort + "/"
 
 ##### Enable dry run to not commit to the database #####
-dryRun = True
+dryRun = False
 verbose = True
+Debug = True
```

Comment:
Is this still needed? it looks like you're just printing some messages for debugging purposes? If you want those to stay, maybe it should just use the `verbose` flag instead.